.github/workflows/ci-actions-incremental.yml

```diff
@@ -42,7 +42,7 @@ concurrency:
 env:
   # Workaround testsuite locale issue
   LANG: en_US.UTF-8
-  COMMON_MAVEN_ARGS: "-e -B --settings .github/mvn-settings.xml --fail-at-end"
+  COMMON_MAVEN_ARGS: "-e -B --settings ${{ github.workspace }}/.github/mvn-settings.xml --fail-at-end"
```

Comment:
Is this change (and all the other similar changes) really needed?

Reply:
This is needed because it will fail some builds due to missing settings.xml. I had this in my ci run.


2022-09-21 20:26:22 (20.9 MB/s) - ‘/home/runner/work/quarkus/quarkus/integration-tests/devtools/target/quarkus-codestart-test/quarkus-jdbc-h2-hibernate-orm-a56eccb5-3f4c-41a1-95ca-1dcadc683908/real-data/java/.mvn/wrapper/maven-wrapper.jar’ saved [59925/59925]

Error:  Error executing Maven.
Error:  The specified user settings file does not exist: /home/runner/work/quarkus/quarkus/integration-tests/devtools/target/quarkus-codestart-test/quarkus-jdbc-h2-hibernate-orm-a56eccb5-3f4c-41a1-95ca-1dcadc683908/real-data/java/.github/mvn-settings.xml

So I avoided it by adding the full path to the argument.

Reply:
Ah, OK, it makes sense then. Thanks for the explanation.